docs/book/features/volume_health.md

```diff
@@ -0,0 +1,85 @@
+# vSphere CSI Driver - Volume Health
+
+Volume Health feature provides the health status of persistent volume. Volume Health is supported for Block Volumes in vSphere with Kubernetes Cluster and Tanzu Kubernetes Grid Cluster - vSphere 7.0u1.
+
+## Health Status of Persistent Volume
+
+Volume Health reports the health status for Persistent Volumes. For each Persistent Volume in `bound` state, the health status of this Persistent Volume will be exposed as an annotation of Persistent Volume Claim which is bound to the Persistent Volume. The key of the annotation is `volumehealth.storage.kubernetes.io/messages:`. There are two possible values for health status.
```

Comment:
Can we add some info on how frequently the volume health is refreshed and what is the max delay in bubbling up a change in volume health from vSphere to PVC?